### ISSUE TITLE
!! undefined is not a function

### ISSUE THREAD

**Issue by ldong** [jdan/cleaver#115]
I use vim to edit my markdowns, after I saved my files using vim.

Cleaver outputs errors like the following:

```bash
cleaver watch demo.md

Watching for changes on demo.md. Ctrl-C to abort.
Rebuilding: Sun Apr 26 2015 15:51:55 GMT-0700 (PDT)
!! undefined is not a function
```

After I turned `--debug` on, like this
```bash
cleaver --debug watch  demo.md                                                                                   

Watching for changes on demo.md. Ctrl-C to abort.
  cleaver loaded input document +0ms
  helper read /usr/local/lib/node_modules/cleaver/templates/layout.mustache +2ms
  helper read /usr/local/lib/node_modules/cleaver/templates/author.mustache +2ms
  helper read /usr/local/lib/node_modules/cleaver/templates/default.mustache +0ms
  cleaver loaded templates +0ms
  cleaver parsed options +4ms
  helper read /usr/local/lib/node_modules/cleaver/resources/default.css +8ms
  helper read /usr/local/lib/node_modules/cleaver/resources/script.js +0ms
  helper read /usr/local/lib/node_modules/cleaver/resources/github.css +0ms
  cleaver loaded static assets +0ms
  cleaver rendered slides +1ms
  cleaver rendered presentation +0ms
Rebuilding: Sun Apr 26 2015 15:54:35 GMT-0700 (PDT)
/usr/local/lib/node_modules/cleaver/lib/index.js:29
    debug.enable('*');
          ^
TypeError: undefined is not a function
    at new Cleaver (/usr/local/lib/node_modules/cleaver/lib/index.js:29:11)
    at /usr/local/lib/node_modules/cleaver/bin/cleaver:21:26
    at fs.js:334:14
    at FSReqWrap.oncomplete (fs.js:95:15)
```

I wonder which part breaks that, thanks.

**Comment by jdan** [comment]
Fixed via #114. Just put out v0.7.4 with the fix - so feel free to update and you should be good to go!

```
npm update -g cleaver
```

Thanks for reporting!

**Comment by ldong** [comment]
Indeed it fixed, thanks.
